contents/product-engineers/multivariate-testing.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Multivariate testing, explained
+date: 2023-11-07
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/posthog-company-culture-blog.png
+featuredImageType: full
+tags:
+  - Guides 
+  - Explainers
+  - Growth engineering
+  - Product engineers
+  - AB testing
+---
+
+## What is multivariate testing
+
+Multivariate testing is when you test different combinations of changes to determine which performs the best. 
+
+For example, you want to test how the color and placement of a button affect your conversion rate. You would then create a test with four different variants:
+
+1. The control variant.
+2. The variant with only the color changed.
+3. The variant with only the placement changed.
+4. The variant with both the color and the placement changed.
+
+The results from the multivariate test show not only which variant performs the best but also how each factor affected the outcome.
+
+It's similar to [A/B testing](/ab-testing), except in A/B testing you're only testing two variants.
+
+![A/B testing vs multivariate testing](../images/blog/multivariated-testing-explained/ab-vs-mvt.png)
+
+## Benefits
+
+### 1. It saves you time
+
+Running multiple A/B tests at the same risks them interfering with each other. As a result, you usually need to wait for each test to finish before starting the next one.
+
+Multivariate tests enable you to test multiple combinations at the same time, meaning you don't have to wait for many consecutive tests to finish.
+
+### 2. It reveals how different elements interact with each other
+
+Multivariate testing is not just about looking at which variant is the best; it's about understanding the weight each factor had in that success. This provides insights into understanding your holistic user experience.
+
+For example, let's say you're testing changes to the user dashboard of a B2B SaaS app. You decide to test how both the position of the navigation menu and the text of the buttons affect user engagement.
+
+In isolation, changing the button texts might affect how well users understand your app. Similarly, moving the navigation menu from the top to the side could influence their ability to navigate it.
+
+However, when you combine and test these changes together, you may discover that their combined effect is not necessarily greater than each individual change.
+
+Multivariate testing enables you to uncover insights like these.
+
+## Drawbacks
+
+### 1. You need a larger sample size
+
+Since you split your participants into many groups, you need a larger sample size for a [statistically significant](/docs/experiments/significance) result. In practice, this means multivariant tests need to for run longer than a typical A/B test.
```

Comment:
```suggestion
Since you split your participants into many groups, you need a larger sample size for a [statistically significant](/docs/experiments/significance) result. In practice, this means multivariate tests need to for run longer than a typical A/B test.
```